lib/CMakeLists.txt

```diff
@@ -150,6 +148,12 @@ if(BUILD_FORTRAN_INTERFACE)
   set_source_files_properties(quda_fortran.F90 PROPERTIES OBJECT_OUTPUTS ${CMAKE_CURRENT_BINARY_DIR}/quda_fortran.mod)
 endif()
 
+
+if(QUDA_TARGET_HIP) 
```

Comment:
we need a different solution here. I don't like start cluttering the CMakeLists with specific workarounds.

Reply:
Any ideas for that already?